src/styles/blocks/card.css

```diff
@@ -0,0 +1,88 @@
+/* #region card */
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+  display: block;
+}
+
+.product-card__name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.product-card__code {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  display: block;
+  color: #616070;
+}
+
+.product-card__rating {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.product-card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.product-card__price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.product-card__price-name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.product-card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.product-card__purchase {
+  box-sizing: border-box;
+  width: 166px;
+  height: 40px;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  background-color: #00acdc;
+  border: 1px solid transparent;
+  border-radius: 5px;
+  text-transform: uppercase;
+  color: #fff;
+}
+
+.product-card__purchase:hover {
+  background-color: #fff;
+  border-color: #00acdc;
+  color: #00acdc;
+}
```

Comment:
I think you can delete borer-color from hover and add instead of transparent this one #00acdc,because its the same as back color.